janitor/functions/pivot.py

```diff
@@ -208,41 +237,66 @@ def pivot_wider(
 
```

Comment:
(this comment pertains to the `pivot_wider` docstring -- not in the diff so I can't comment on it directly in the review)

I think, in general, whenever we refer to our own functions, we should add links so that the user can click on them to jump to it. See [here](https://github.com/pyjanitor-devs/pyjanitor/blob/fd59453d6109a9198ad7e79b95e96344d4d0a014/janitor/functions/deconcatenate_column.py#L35) and view the result on the [docs](https://pyjanitor-devs.github.io/pyjanitor/api/functions/#janitor.functions.deconcatenate_column.deconcatenate_column), for an example on how to do this.

To be explicit, in this docstring, we refer to `pivot_longer` and `select_columns`; I think we can add links to these.